packages/orchestration/src/exos/local-orchestration-account.js

```diff
@@ -207,9 +215,9 @@ export const prepareLocalChainAccountKit = (
          * updater will get a special notification that the account is being
          * transferred.
          */
-        /** @type {LocalChainAccount['deposit']} */
-        async deposit(payment, optAmountShape) {
-          return V(this.state.account).deposit(payment, optAmountShape);
+        /** @type {OrchestrationAccount<any>['deposit']} */
+        async deposit(payment) {
+          await V(this.state.account).deposit(payment);
```

Comment:
In a future PR we might consider updating `OrchestrationAccountI['deposit']` to return `Promise<Amount<'nat'>>`. I don't see a reason to swallow that response.

Tangentially - with the revised implementation plan for #9193 only `LocalOrchestrationAccount` will have `.deposit()`.